_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: TODO
+exercises: TODO
+questions:
+- "TODO"
```

Comment:
* How can I parallelize computations on rasters with dask?
* How can I determine if parallelization improves calculation speed?
* How can I control how much to parallelize calculations?